src/diffusers/pipelines/allegro/pipeline_allegro.py

```diff
@@ -198,6 +198,10 @@ def __init__(
         scheduler: KarrasDiffusionSchedulers,
     ):
         super().__init__()
+        self._guidance_scale = 1.0
```

Comment:
i would suggest:

1 .default _guidance_scale to be same as each pipeline __call__ default, e.g. 7.5 here

2 _num_stesps can be None here since it is not yet known 


but let me know all your thoughts @DN6 @ishan-modi